comfyapp.py

```
@@ -540,6 +676,11 @@ def link_directory(target: Path, source: Path) -> bool:
         if link_directory(comfy_root / "user", USER_DATA_VOLUME_MOUNT):
             print(f"{comfy_root} の user ディレクトリを永続化 Volume に接続しました")
 
+        # config.ini は user/ 配下なので、Volume へ繋いだ後に書く。
+        # 値は Secret 経由で注入されるため、モジュール読み込み時ではなく
+        # ここで読み直す（import と Secret 注入の順序に依存しないため）。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)